R/zzz.R

```diff
@@ -1,6 +1,14 @@
 
 # Create the log environment on load
 .onLoad <- function(libname, pkgname) {
+   # init functions for other timber functionality
    log_init()
    approved_functions_init()
+
+   # This overwrite is to correctly build purrr adverb function
+   # outlined in purrr best practices for exporting adverb-wrapped functions
+   run_safely <<- purrr::safely(run_file, quiet = FALSE)
```

Comment:
is the `purrr::` necessary?

Reply:
in `zzz.R` you usually use the full call including package from my experience.

Reply:
cool!  I have no experience with zzz.R so I defer to your experience! :)